x-pack/plugins/cases/public/application.tsx

```diff
@@ -65,6 +65,7 @@ export const App: React.FC<{ deps: RenderAppProps }> = ({ deps }) => {
           kibanaVersion,
           ...coreStart,
           ...pluginsStart,
+          securityService: coreStart.security,
```

Comment:
Note, `KibanaContextProvider` isn't well typed for custom service objects like this. See the FIXME in `x-pack/plugins/cases/public/common/lib/kibana/kibana_react.ts`